docs/docs/sourcing-from-graphcms.md

```diff
@@ -0,0 +1,162 @@
+
+### Add the Source Plugin
+
+Additionally, you need the the `gatsby-source-graphql` library. Because GraphCMS uses GraphQL natively, we will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
+
+You can install this component with:
+
+```bash
+  # Optionally with `npm install`
+  yarn add gatsby-source-graphql
+```
+
+### Configure the Plugin
+
+The last step required before you can query your data is to configure the `gatsby-source-graphql` plugin. Open `gatsby-config.js` and add the following object to the plugins array. This example uses an open API from GraphCMS but you will most likely want to replace this with your own API and provide a fieldName that makes the most sense to your project.
+
+```js
+{
+    resolve: "gatsby-source-graphql",
+        options: {
+        // The top level query type, can be anything you want!
+        typeName: "GCMS",
+        // The field you'll query against, can also be anything you want.
+        fieldName: "gcms",
+        // Your API endpoint, available from the dashboard and settings window.
+        // You can use this endpoint that features US mountains for now.
+        url: "https://api-euwest.graphcms.com/v1/cjm7tab4c04ro019omujh708u/master",
+    },
+},
+```
+
+If everything works correctly, you should now have your GraphCMS data added to the Gatsby source API!
+
+### Querying for Content
+
+From the root of your project, run the development environment with `yarn run development`. Once the server has started and is error free, you should be able to open the following URL in your browser:
+
+http://localhost:8000/___graphql
+
+This will show you a "Graphiql" playground where you can test your new content API.
+
+Try running this query:
+
+```js
+query {
+    gcms {
+        mountains {
+            title
+            elevation
+            image {
+                url
+            }
+        }
+    }
+}
+```
+
+Again, if everything is working properly, you should see a successful response in the shape of:
+
+```json
+{
+  "data": {
+    "gcms": {
+      "mountains": [
+        {
+          "title": "Denali",
+          "elevation": 6190,
+          "image": {
+            "url": "https://media.graphcms.com//J0rGKdjuSwCk7QrFxVDQ"
+          }
+        },
+        {
+          "title": "Mount Elbert",
+          "elevation": 4401,
+          "image": {
+            "url": "https://media.graphcms.com//JNonajrqT6SOyUKgC4L2"
+          }
+        }
+        // ...more results
+      ]
+    }
+  }
+}
+```
+
+### Getting Content on the Page
+
+For the purpose of this tutorial I've removed all the layout, SEO, link or other special components that comprise a page in the Gatsby starter. Open the index file located at `src/pages/index.js` and replace the content with this code:
```

Comment:
You could use `gatsby-starter-hello-world` instead, if you don't want to deal with any of the peripheral pieces of the default starter.

Reply:
I actually like leaving it in for people to have access to it as they'll likely want it for their projects and a lot of examples reference it. I just wanted to emphasize the relevant code. What do you think?

Reply:
Just adding a quick ping @jlengstorf

Reply:
Maybe the wording is unclear, then? I thought you meant that you had installed the default starter, then deleted the code you weren't going to use. Do you mean that you're just omitting it in the code sample?

I think I'm unclear on the intention.

Reply:
Gotcha, I'll fix.